### ISSUE TITLE
Missed required argument stockConfig in parent::__construct call

### ISSUE THREAD

**Issue by ShapesGraphicStudio** [interjar/configurable-child-visibility#17]
Hello,

I'm trying to get this module working with Magento 2.3.4.

When trying to setup:di:compile I get: 
```
Errors during compilation:
Interjar\ConfigurableChildVisibility\Plugin\Model\ResourceModel\Attribute\InStockOptionSelectBuilder
Missed required argument stockConfig in parent::__construct call
```

Any idea on how to get this fixed please?

**Comment by adamj88** [comment]
I think that plugin is redundant in 2.3.4 as it was fixed in: https://github.com/magento/magento2/commit/dd761d02d824aaa7666d8875d7081ad124a6aabe#diff-56a04bb15a75c59fb3261f479e2e2e30

Will need to investigate how that commit impacts the rest of the codebase.

**Comment by ShapesGraphicStudio** [comment]
Hi, thanks for you answer.
Using Magento 2.3.4 as I said, the out of stock children products do not display on configurable product page.
Configuration > Catalog > Inventory > Display Out of Stock Products is set to Yes.

**Comment by danbtl** [comment]
@ShapesGraphicStudio Were you able to find a way to have swatches of out of stock items clickable?

**Comment by nix0x00** [comment]
I am getting similar error on Magento 2.3.5-p1, reason I am trying to install is because I cannot get out of stock (configurable product) crossed out.

Error:

```
Errors during compilation:
        Interjar\ConfigurableChildVisibility\Plugin\Model\ResourceModel\Attribute\InStockOptionSelectBuilder
                Missed required argument stockConfig in parent::__construct call. File: /var/www/magento/vendor/interjar/module-configurable-child-visibility/Plugin/Model/ResourceModel/Attribute/InStockOptionSelectBuilder.php
Total Errors Count: 1
```

**Comment by adamj88** [comment]
Resolved in #19
